.github/workflows/ci.yml

```diff
@@ -45,6 +45,9 @@ env:
   PERCY_TOKEN_PIE_TEXTAREA: ${{ secrets.PERCY_TOKEN_PIE_TEXTAREA }}
   PERCY_TOKEN_PIE_TOAST: ${{ secrets.PERCY_TOKEN_PIE_TOAST }}
   TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
+  PERCY_DO_NOT_CAPTURE_RESPONSIVE_ASSETS: true
```

Comment:
Why did you need to add this?

Reply:
It may have been causing some of the request issues we were seeing in some of our builds and was suggested by the Percy team to add it in as part of the troubleshooting steps.